src/data/__tests__/calculate.test.ts

```diff
@@ -544,4 +545,86 @@ describe('calculate', () => {
       ).toEqual(defaultValue * 2 * 0.1)
     })
   })
+
+  describe('percentFullyVaccinated', () => {
+    const defaultScenario = {
+      ...testData(prepopulated['1person_15minCarRide']),
+      averageFullyVaccinatedMultiplier: 0.4,
+      percentFullyVaccinated: 40,
+      unvaccinatedPrevalenceRatio: null,
+    }
+    const defaultUnvaccinatedPrevalenceRatio = 1 / (0.4 * 0.4 + 0.6)
+
+    it.each`
+      theirVaccine
+      ${'vaccinated'}
+      ${'unvaccinated'}
+    `(
+      'Should decrease risk for lower vaccination rate when interacting with $theirVaccine people',
+      ({ theirVaccine }) => {
+        const defaultValue = calcValue({ ...defaultScenario, theirVaccine })!
+        const value = calcValue({
+          ...defaultScenario,
+          theirVaccine,
+          percentFullyVaccinated: defaultScenario.percentFullyVaccinated! / 2,
+        })
+        const expectedUnvaccinatedPrevalenceRatio = 1 / (0.4 * 0.2 + 0.8)
+        expect(value).toBeCloseTo(
+          defaultValue *
+            (expectedUnvaccinatedPrevalenceRatio /
+              defaultUnvaccinatedPrevalenceRatio),
+        )
+      },
+    )
+
+    it.each`
+      theirVaccine
+      ${'vaccinated'}
+      ${'unvaccinated'}
+    `(
+      'Should increase risk for higher vaccination rate when interacting with $theirVaccine people',
```

Comment:
Personally I would have done this as a multi-column it.each, but I don't insist.